doc/release-notes-27600.md

```diff
@@ -0,0 +1,10 @@
+P2P
+---
+
+- A new net permission `ForceInbound` (set with `-whitelist=forceinbound@...`
+  or `-whitebind=forceinbound@...`) is introduced that extends `NoBan` permissions.
+  Inbound connections from hosts protected by `ForceInbound` will now be more
```

Comment:
nit: the capitalized spelling is an internal implementation detail, would stick to `forceinbound` and `noban` here
```suggestion
- A new net permission `forceinbound` (set with `-whitelist=forceinbound@...`
  or `-whitebind=forceinbound@...`) is introduced that extends `noban` permissions.
  Inbound connections from hosts protected by `forceinbound` will now be more
```

Reply:
done, thanks